features/support/env.js

```diff
@@ -1,20 +1,33 @@
-const { setWorldConstructor, After, setDefaultTimeout } = require('cucumber');
+const { setWorldConstructor, After, setDefaultTimeout, Status } = require('cucumber');
 const { Builder } = require('selenium-webdriver');
 const firefox = require('selenium-webdriver/firefox');
 
 class CustomWorld {
-  constructor() {
+  constructor({ attach }) {
+    this.attach = attach;
     this.driver = new Builder()
       .forBrowser('firefox')
-      .setFirefoxOptions(
-        new firefox.Options().headless()
-      ).build();
+      .setFirefoxOptions(this.firefoxOption())
+      .build();
+  }
+
+  firefoxOption() {
+    // GitHub Actions set CI to true
+    // Ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
+    return process.env.CI ? new firefox.Options().headless() : new firefox.Options()
   }
```

Comment:
I don't remember these changes being made in our mob testing session. Aren't these from https://github.com/zinc-collective/convene/pull/146/files ? Should those changes be included in this PR?

Reply:
Yes, I split out https://github.com/zinc-collective/convene/pull/146/ intentionally so those code will be in its own PR but I need these code to drive today session so I don't need to manually toggle headless mode.
And yes, it will be removed from this PR.